public/docs/ts/latest/glossary.jade

```diff
@@ -54,7 +54,7 @@ include _util-fns
     The barrel itself is a module file that re-exports *selected* exports of other modules.
     
     Imagine three modules in a `heroes` folder:
-  code-example(format='').
+  code-example.
```

Comment:
Actually, something has changed. `code-example` ***used to*** display line numbers by default. 

Now the default is *no line numbers*. If you want them you have to add them explicitly as in:
```
code-example(format="linenums").
  // Code with line numbers
  one
  3
  4
```
I wonder when this change took place? Not that I'm objecting.